web/src/app/services/ServiceRules/ServiceRuleForm.tsx

```diff
@@ -300,6 +300,53 @@ export default function ServiceRuleForm(
     )
   }
 
+  const renderContents = (
+    actionIdx: number,
+    action: ServiceRuleActionInput,
+  ): React.ReactNode => {
+    return action.contents.map((c: Content, contentIdx: number) => {
+      if (
+        (action.destType === destType.SLACK && c.prop === 'channel') ||
+        (action.destType === destType.SLACK && c.prop === 'channel_id')
+      ) {
+        return <React.Fragment key={contentIdx} />
+      }
+      return (
+        <Grid
+          key={c.prop}
+          item
+          style={{ flexGrow: 1, marginTop: '1em' }}
```

Comment:
nit: use the sx prop on MUI components moving forward. It should work the same but uses mui's newer `@emotion` code instead
```suggestion
          sx={{ flexGrow: 1, marginTop: '1em' }}
```